fam/llm/fast_inference_utils.py

```diff
@@ -359,18 +359,18 @@ def build_model(
 
     device_sync(device=device)  # MKG
     t0 = time.perf_counter()
-    y = generate(
-        model,
-        encoded,
-        spk_emb,
-        max_new_tokens=200,
-        callback=lambda x: x,
-        temperature=torch.tensor(1.0, device=device, dtype=precision),
-        top_k=None,
-        top_p=torch.tensor(0.95, device=device, dtype=precision),
-        guidance_scale=torch.tensor(3.0, device=device, dtype=precision),
-        end_of_audio_token=9999,  # don't end early for compilation stage.
-    )
+    # y = generate(
```

Comment:
Why comment this out? How do you test the model without this?

Reply:
I'll add this back in, this was for faster initialization on my local setup (slow GPU)